source/installer.py

```diff
@@ -621,9 +629,27 @@ def tryCopyFile(sourceFilePath,destFilePath):
 			errorCode = ctypes.GetLastError()
 			raise OSError("Unable to copy file %s to %s, error %d"%(sourceFilePath,destFilePath,errorCode))
 
+_nvdaExes = {
+	"nvda.exe",
+	"nvda_noUIAccess.exe",
+	"nvda_uiAccess.exe",
+	"nvda_dmp.exe",
+	"nvda_slave.exe"
+}
+
+
+def _isNVDAAlive(installDir: str) -> bool:
+	fullPathNvdaExes = {os.path.join(installDir, fn) for fn in _nvdaExes}
+	for process in psutil.process_iter():
+		if process.exe() in fullPathNvdaExes:
+			return True
+	return False
+
 
 def install(shouldCreateDesktopShortcut: bool = True, shouldRunAtLogon: bool = True):
 	prevInstallPath=getInstallPath(noDefault=True)
+	if _isNVDAAlive(prevInstallPath):

```

Comment:
This should handle  a case where NVDA was never installed on a given system:
```suggestion
	if prevInstallPath is not None and _isNVDAAlive(prevInstallPath):
```